src/hb-open-file.hh

```diff
@@ -119,10 +119,10 @@ typedef struct OpenTypeOffsetTable
 
   public:
 
-  template <typename item_t>
+  template <typename Iterator>
```

Comment:
Can you maybe add a `hb_is_iterator_of` qualifier here?

Reply:
tried iterator_of but that failed, I suspect similar issue to the map default value parameter. Using hb_is_iterator for now.

Reply:
Right, that had some problems. I'll see if I can fix.